scripts/install_dgraph.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash
-curl https://get.dgraph.io -sSf | bash
+
+curl https://get.dgraph.io -sSf | bash -s -- -y
```

Comment:
Not sure if we should accept the license without user seeing the license. Is that alright?

Reply:
Actually this is to run at our end. Mean travis-ci.